bdio-tinkerpop/src/main/java/com/blackducksoftware/bdio2/tinkerpop/spi/BlackDuckIoReaderSpi.java

```diff
@@ -135,7 +135,16 @@ protected void getNodeProperties(Map<String, Object> node, BiConsumer<Object, Ob
     protected void getNodeEdges(Map<String, Object> node, BiConsumer<String, Object> edges) {
         node.forEach((term, value) -> {
             if (frame().isObjectProperty(term)) {
-                edges.accept(term, frame().context().fromFieldValue(term, value));
+
+                if (value instanceof List<?>) {
```

Comment:
We need to make sure to test very carefully these changes in Black Duck for many Protex BDIO, Dependency BDIO, and Binary Analysis BDIO before fully including this library change into Black Duck.   I would highly recommend informing QA (Mike Regan) to form a specific test plan around this.